components/crud-web-apps/jupyter/manifests/overlays/aws/config-map.yaml

```diff
@@ -0,0 +1,143 @@
+apiVersion: v1
```

Comment:
Should the AWS overlay be placed here? How will this work with OWNERS?

Reply:
That's a good question. AFAIK, distribution specific overlays make more sense to exist in the distribution repos.
But for this PR, I would suggest to leave it out of scope and focus on simply transferring back what kubeflow/manifests already has.